compiler/modules/front.syntax/ast.bal

```diff
@@ -55,14 +55,19 @@ public type ConstDefn record {|
 |};
 
 public type Stmt VarDeclStmt|AssignStmt|CallStmt|ReturnStmt|IfElseStmt|MatchStmt|WhileStmt|ForeachStmt|BreakContinueStmt|CompoundAssignStmt|PanicStmt;
-public type CallStmt FunctionCallExpr|MethodCallExpr|CheckingStmt;
-public type Expr NumericLiteralExpr|ConstValueExpr|FloatZeroExpr|VarRefExpr|CompoundExpr;
+public type CallExpr FunctionCallExpr|MethodCallExpr|CheckingCallExpr;
+public type Expr NumericLiteralExpr|ConstValueExpr|FloatZeroExpr|VarRefExpr|CompoundExpr|CallExpr;
```

Comment:
CheckingCallExpr is a subtype of CheckingExpr, so this is including CheckingCallExpr in Expr twice (once via CallExpr and once via CheckingExpr). So better to use `FunctionCallExpr|MethodCallExpr` instead of `CallExpr` in definition of `Expr`. This matches the grammar in the spec.

Reply:
fixed